fusedrug/data/tokenizer/modulartokenizer/test_multi_tokenizer_use.py

```diff
@@ -18,9 +18,7 @@ def update_special_tokens(
     return tokenizer_inst
 
 
-@hydra.main(
-    config_path="./configs", config_name="tokenizer_config_personal", version_base=None
-)
+@hydra.main(config_path="./configs", config_name="tokenizer_config", version_base=None)
```

Comment:
Wasn't sure if that's intentional or not, but this change was needed for it to run